### PR TITLE
New teacher email fixes

### PR DESCRIPTION
**Pull request by ewjordan**
1) Changes the form-based "Get started!" button to a more sane normal href so behavior is less odd

2) Shows some US-only text for those with US IP addresses

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Am I right in thinking that we can't use the associated `UserGeo` because it gets populated too late (though the cron runs every minute, it is not guaranteed to geocode all uncoded users)? It may also be created too late in the "new teacher workflow".

File: dashboard/app/models/user.rb [hunk 1]
```diff
@@ -328,6 +328,7 @@ def district_teachers(cohort = nil)
     ['gender.none', '-']
   ].freeze
 
+  attr_accessor :ip
```
Comment (ashercodeorg) [inline dashboard/app/models/user.rb]: If necessary, this should be (well) documented. Is there a reason not to use `User.current_sign_in_ip`?

**Comment by ewjordan** [comment]
Yes, this action triggers immediately upon account creation, so the record is not populated yet.

File: dashboard/test/controllers/registrations_controller_test.rb [hunk 2]
```diff
@@ -143,8 +143,31 @@ class RegistrationsControllerTest < ActionController::TestCase
     assert_equal ["Age is required"], assigns(:user).errors.full_messages
   end
 
-  test "create new teacher sends email" do
+  test "create new teacher with us ip sends email with us content" do
     teacher_params = @default_params.update(user_type: 'teacher')
+    Geocoder.stubs(:search).returns(
+      [OpenStruct.new(
+        country_code: 'US'
+      )]
```
Comment (ashercodeorg) [inline dashboard/test/controllers/registrations_controller_test.rb]: Any reason not to one-line this?
```
    Geocoder.stubs(:search).returns([OpenStruct.new(country_code: 'US')])
```

**Pull request merged**